modules/oadp-restore-request-limit.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: CONCEPT
+[id="oadp-restore-request-limit_{context}"]
+= Pods crash or restart due to lack of memory or CPU
+
+You can set specific resource requests for memory or CPU if a Velero or Restic pod crashes due to a lack of either of them.
+
+== Velero pod
+
+Use the `configuration.velero.podConfig.resourceAllocations` specification field in the `oadp_v1alpha1_dpa.yaml` file to set specific resource requests for a Velero pod.
+
+.Example Velero `yaml` file:
+
+[source,yaml]
+----
+  configuration:
+    velero:
+      podConfig:
+        resourceAllocations:
+          requests:
+            cpu: 500m
+            memory: 256Mi
+----
+
+== Restic pod
+
+Use the `configuration.restic.podConfig.resourceAllocations` specification field to set specific resource requests for a Restic pod.
+
+.Example Restic `yaml` file:
+
+[source,yaml]
+----
+  configuration:
+    restic:
+      podConfig:
+        resourceAllocations:
+          requests:
+            cpu: 500m
+            memory: 256Mi
+----
+
+[NOTE]
+====
+The value for the resource requests flags follows the same format as Kubernetes resource requirements.
+
+Also, if you do not define `configuration.velero.podConfig.resourceAllocations` or `configuration.restic.podConfig.resourceAllocations`, then the default resources specification for a Velero or a Restic pod is:
```

Comment:
@RichardHoch 
I would suggest:
"Also, if you do not specify `configuration.velero.podConfig.resourceAllocations` or `configuration.restic.podConfig.resourceAllocations`, the default `resources` specification for a Velero or a Restic pod is as follows:"

Reply:
+1